fast64_internal/oot/scene/exporter/to_c/actor.py

```diff
@@ -9,45 +9,38 @@
 # Actor List
 
 
-def ootActorToC(actor):
+def getActorEntry(actor: OOTActor):
+    """Returns a single actor entry"""
+    posData = "{ " + ", ".join([f"{round(pos)}" for pos in actor.position]) + " }"
+    rotData = "{ " + "".join(rot for rot in actor.rotation) + " }"
+
+    actorInfos = [actor.actorID, posData, rotData, actor.actorParam]
+    infoDescs = ["Actor ID", "Position", "Rotation", "Parameters"]
+
     return (
-        "{ "
-        + ", ".join(
-            (
-                str(actor.actorID),
-                str(int(round(actor.position[0]))),
-                str(int(round(actor.position[1]))),
-                str(int(round(actor.position[2]))),
-                *(
-                    (
-                        actor.rotOverride[0],
-                        actor.rotOverride[1],
-                        actor.rotOverride[2],
-                    )
-                    if actor.rotOverride is not None
-                    else (
-                        str(int(round(actor.rotation[0]))),
-                        str(int(round(actor.rotation[1]))),
-                        str(int(round(actor.rotation[2]))),
-                    )
-                ),
-                str(actor.actorParam),
-            )
-        )
-        + " },\n"
+        (f"// {actor.actorName}\n" + indent if actor.actorName != "" else "")
+        + "{\n"
+        + ",\n".join((indent * 2) + f"/* {desc:10} */ {info}" for desc, info in zip(infoDescs, actorInfos))
```

Comment:
I'm not sure if comments like this break scene importing, have you tried it out?

Reply:
I tried it with vanilla scenes and it worked, I forgot you can import fast64 scenes now lol (also does it work properly with non-single file mode?)

Reply:
(it doesn't seem to work with non-single file mode)

Reply:
just in case, I'm working on a fix for this

Reply:
fast64 scenes import should now be fixed, idk how to check if everything works for sure but I fixed every errors

something better can be used to detect if light settings are using the new format but I wasn't inspired lol, though the multiple files issue isn't fixed as it wasn't brought by this PR (and I have no idea how to fix that)

anyway, tell me if anything else broke 👍